package-lock.json

```diff
@@ -33824,12 +33823,12 @@
 			}
 		},
 		"eslint-plugin-jest": {
-			"version": "25.2.3",
-			"resolved": "https://registry.npmjs.org/eslint-plugin-jest/-/eslint-plugin-jest-25.2.3.tgz",
-			"integrity": "sha512-Yoa0at3euTjERDvPGPWiItY1uuqKYQ5Ov2SmkSLmKRq9OFiVdEehw0rWuK4PA538k7CNqnvmkztjAB9l+HJ7kQ==",
+			"version": "27.2.1",
```

Comment:
This is expected: we're not requiring the minimum version that `eslint-plugin` requires.